src/components/Player/ClueText.js

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 
+function decodeHtml(htmlText) {
```

Comment:
any reason not to use this? https://lodash.com/docs/4.17.15#unescape

Reply:
I tried that and it wouldn't unescape the bulk of anything in the test puzzle mentioned in #220. I found this explanation: https://stackoverflow.com/questions/36352155/unescape-function-from-underscore-js-not-working

Reply:
https://github.com/mathiasbynens/he would also be an option, but not sure whether bringing a library in for this is appropriate. (As in genuinely don't know, pretty new to JS development 😄 )

Reply:
gotcha! i think importing he or doing it as you did are both reasonable! thanks for the explanation